### PR TITLE
aom: build shared libs

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/aom.rb [hunk 1]
```diff
@@ -28,7 +29,8 @@ def install
                                     "-DENABLE_EXAMPLES=on",
                                     "-DENABLE_TESTDATA=off",
                                     "-DENABLE_TESTS=off",
-                                    "-DENABLE_TOOLS=off"])
+                                    "-DENABLE_TOOLS=off",
+                                    "-DBUILD_SHARED_LIBS=1"])
```
Comment (SMillerDev) [inline Formula/aom.rb:33]:
```suggestion
                                    "-DBUILD_SHARED_LIBS=on"])
```
Should this be like the rest?

**Comment by iMichka** [comment]
```
2021-01-05T22:35:23.5204050Z [32m==>[0m [1mTesting aom[0m
2021-01-05T22:35:23.5206750Z /usr/bin/sandbox-exec -f /private/tmp/homebrew20210105-60015-ln8evd.sb ruby -W0 -I $LOAD_PATH -- /opt/homebrew/Library/Homebrew/test.rb /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/aom.rb --verbose
2021-01-05T22:35:23.5210700Z cp -p /Users/brew/Library/Caches/Homebrew/downloads/ced756f9d1f53c598f4f2b5c3676f9a313d4102ac938a50a36d66ea7343e22bd--bus_qcif_15fps.y4m /private/tmp/aom--bus_qcif_15fps.y4m-20210105-60016-oanx08/bus_qcif_15fps.y4m
2021-01-05T22:35:23.5213980Z [34m==>[0m [1m/opt/homebrew/Cellar/aom/2.0.1_1/bin/aomenc --webm --tile-columns=2 --tile-rows=2 --cpu-used=8 --output=bus_qcif_15fps.webm bus_qcif_15fps.y4m[0m
2021-01-05T22:35:23.5215690Z dyld: Library not loaded: @rpath/libaom.2.dylib
2021-01-05T22:35:23.5216860Z   Referenced from: /opt/homebrew/Cellar/aom/2.0.1_1/bin/aomenc
2021-01-05T22:35:23.5218150Z   Reason: image not found
2021-01-05T22:35:23.5219940Z /opt/homebrew/Library/Homebrew/shims/scm/git --version
2021-01-05T22:35:23.5221160Z /usr/bin/curl --version
2021-01-05T22:35:23.5222410Z /usr/libexec/java_home --xml --failfast
2021-01-05T22:35:23.5223680Z [31mError:[0m aom: failed
```

**Comment by carlocab** [comment]
```
==> brew audit aom --online --git --skip-style
==> FAILED
aom:
  * Stable resource "bus_qcif_15fps.y4m": The URL https://media.xiph.org/video/derf/y4m/bus_qcif_15fps.y4m is not reachable
Error: 1 problem in 1 formula detected
```
Only happened on Big Sur, so probably just a transient failure.

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
@iMichka it seems your experimenting with the cellar somehow made it into this PR?

https://github.com/Homebrew/homebrew-core/pull/67582/commits/8b1dbfef80b6e696b5f4e2e82219733be05d9321

**Comment by iMichka** [comment]
> @iMichka it seems your experimenting with the cellar somehow made it into this PR?

Yes, sorry. I removed that.

**Comment by carlocab** [comment]
You'll need to delete the duplicate bottles from Bintray first...

**Comment by iMichka** [comment]
Done

**Pull request closed without merging**